src/components/editor.tsx

```diff
@@ -237,7 +233,7 @@ export default function Editor({
         }}
       >
         <h2>Challenge</h2>
-        <Button onClick={(e) => unlockAll()}>Unlock all</Button>
```

Comment:
`unlockAll` has not parameters so we can supply it directly to the `onClick` props